test/conformance/OWNERS

```diff
@@ -4,9 +4,11 @@
 reviewers:
   - cheftako
   - oomichi
+  - johnbelamaric
 approvers:
   - cheftako
   - spiffxp
+  - johnbelamaric
```

Comment:
On https://github.com/kubernetes/kubernetes/commit/8ee8b77eed73cd3f39efb84bee24ee9d6447a3db#diff-fcbdff38f489e333841efa50e8f641f5 @spiffxp took a steady step to an approver from a reviewer.
I am not sure this approver promotion in a single PR according to the above PR.